w_common/lib/src/common/cache/cache.dart

```diff
@@ -150,22 +150,22 @@ class Cache<TIdentifier, TValue> extends Object with Disposable {
 
   /// The stream of [CacheContext]s that dispatches when an item is updated in
   /// the cache.
-  Stream<CacheContext<TIdentifier, TValue>> get didUpdate =>
+  Stream<CacheContext<TIdentifier, TValue?>> get didUpdate =>
       _didUpdateController.stream;
 
   /// Keys that have not been released.
   Iterable<TIdentifier> get liveKeys =>
-      _cache.keys.where((TIdentifier key) => !_isReleased[key]);
+      _cache.keys.where((TIdentifier key) => !_isReleased[key]!);
```

Comment:
I think it would be better to assume a key not in the map has not been release, then throw.
```suggestion
      _cache.keys.where((TIdentifier key) => !(_isReleased[key] ?? false));
```